pkg/agent/api/debug/v1/service_test.go

```diff
@@ -35,7 +35,8 @@ func TestGetInfo(t *testing.T) {
 	// Create root CA
 	ca := testca.New(t, td)
 	cachedBundleCert := ca.Bundle().X509Authorities()[0]
-	cachedBundle := bundleutil.BundleFromRootCA("spiffe://example.org", cachedBundleCert)
+	trustDomain := spiffeid.RequireTrustDomainFromString("spiffe://example.org")
```

Comment:
nit: even though RequireTrustDomainFromString can parse the trust domain out of the ID, i think it is clearer in intent to just pass the trust domain:
```suggestion
	trustDomain := spiffeid.RequireTrustDomainFromString("example.org")
```